libretroshare/src/jsonapi/jsonapi.cpp

```diff
@@ -144,6 +144,9 @@ bool RsJsonApi::parseToken(
 	for(uint32_t i=0; i < tkLen; ++i)
 		if(clear_token[i] == ':') { colonIndex = i; break; }
 
+	if (colonIndex == std::string::npos)
+		return false;
```

Comment:
a token without ":" means that is a password-less user which is acceptable as per HTTP authentication and also explicitly asked by few people, so please remove this check

Reply:
Will do

Reply:
done